tests/data/pronounce/litexa/main.litexa

```diff
@@ -25,3 +21,48 @@ TEST "say no"
   alexa: launch, /I really like my cat/i
   user: AMAZON.NoIntent
   alexa: null, /goodbye then/i
+
+TEST "say yes en-CA"
```

Comment:
Could you update these tests to also regex check that the phoneme pronunciation is being applied to all languages except for en-IN?

Reply:
Could you be more specific? I do `alexa: null, /phoneme/` in all tests except for en-IN.